lua/entities/gmod_wire_expression2/base/preprocessor.lua

```diff
@@ -44,7 +44,7 @@ end
 function PreProcessor:FindComments(line)
 	local ret, count, pos, found = {}, 0, 1, nil
 	repeat
-		found = line:find('[#"]', pos)
+		found = line:find('[#"\\\\]', pos)
```

Comment:
Usually you'd need to escape the backslash in a regex, but I forgot that in Lua patterns, the escape character is %, not \\.